docs/content/release_notes/csi_rn_whatsnew.md

```diff
@@ -1,12 +1,22 @@
-# What's new in 1.6.0
+# What's new in 1.7.0
 
-IBM® block storage CSI driver 1.6.0 introduces the enhancements detailed in the following section.
+IBM® block storage CSI driver 1.7.0 introduces the enhancements detailed in the following section.
 
-**General availability date**: 18 June 2021
+**General availability date**: 30 September 2021
+
+## Now supports CSI Topology
+
+IBM® block storage CSI driver 1.7.0 is now topology aware.  Using this feature, volume access can be limited to a subset of nodes, based on regions and availability zones. Nodes may be located in various regions within an availability zone, or across the different availability zones. Using CSI Topology feature, can ease volume provisioning for workloads within a multi-zone architecture.
```

Comment:
`Using CSI Topology feature, can ease volume provisioning...` the comma here looks redundant to me